src/content/api/hot-module-replacement.md

```diff
@@ -90,6 +91,82 @@ module.hot.dispose(data => {
 ```
 
 
+### `invalidate`
+
+Calling this method will invalidate the current module which disposes and recreates this module when the HMR update is applied. This bubbles like a normal update of this module. `invalidate` can't be self-accepted by this module.
+
+When called during the `idle` state, a new HMR update will be created containing this module. HMR will enter the `ready` state.
+
+When called during the `ready` or prepare `state`, this module will be added to the current HMR update.
+
+When called during the `check` state, the module will be added to the update when an update is available. If no update is available it will create a new update. HMR will enter the `ready` state.
```

Comment:
Consistency:
```suggestion
When called during the `check` state, this module will be added to the update when an update is available. If no update is available it will create a new update. HMR will enter the `ready` state.
```